soroban-synth-wasm/src/mod_emitter.rs

```diff
@@ -56,6 +56,12 @@ pub struct ModEmitter {
     import_refs: HashMap<(String, String, Arity), FuncRef>,
 }
 
+impl Default for ModEmitter {
```

Comment:
That's weird, why is this needed?

Reply:
It's a suggestion. Since `new` doesn't take any arguments it's akin to `default`. Can suppress this one if it is unwanted.

Reply:
Sure, I realize it's a suggestion. I'm just trying to understand which diagnostics don't seem to be useful. IMHO this one is not super useful, especially given that host implementation is pretty self-contained. It feels a bit weird to introduce custom trait implementations that aren't actually being used anywhere.

Reply:
Gone!